web/client/src/components/soro-select.vue

```diff
@@ -0,0 +1,68 @@
+<template>
+    <div class="material-select">
+        <select
+            class="material-select-text"
+            @change="emitChange"
+        >
+            <option
+                v-for="option in extendedOptions"
+                :key="option"
+                :value="option"
+                :selected="option === value"
+                :label="option ?? ''"
+            >
+                {{ option }}
```

Comment:
Still fucked highlighting

Reply:
I have no idea why this is displayed incorrectly. It is only an issue here on github and not locally. Cant do anything really about it.

Also please stop saying "fucked" everytime something looks a bit wonky.